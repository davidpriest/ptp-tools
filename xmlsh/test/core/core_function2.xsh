# Test of function declarations


args ()
{
	echo args are $*
}

setvar ()
{
	_A=$1
}

subpipe() 
{
	echo <[<foo>bar</foo>]> | xcat
}


echo Test of Args
args arg1 arg2 arg3

echo Test of setvar
setvar ABC
echo _A is $_A

echo Test of subpipe

subpipe | xcat 

# Functions returning values

ret0()
{
  echo in ret 0 ;
  return 0;
  echo SNH
}

ret0 && echo Success of ret0
ret0 || echo Fail of ret0 

