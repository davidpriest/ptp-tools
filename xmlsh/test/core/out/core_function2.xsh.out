Test of Args
args are arg1 arg2 arg3
Test of setvar
_A is ABC
Test of subpipe
<foo>bar</foo>
in ret 0
Success of ret0
in ret 0
